### ISSUE TITLE
rxDataStep with data frame input and HDFS output in remote Spark CC hangs R

### ISSUE THREAD

**Issue by hongooi73** [RevolutionAnalytics/dplyrXdf#41]
```r
hd <- RxHdfsFileSystem()
mthc <- RxXdfData("/user/sshuser/mtcarsc", fileSystem=hd, createCompositeSet=TRUE)
rxDataStep(mtcars, mthc)
#Warning: System error: Attempting to load HDFS functions in unsupported environment. (Failed to load C:/PROGRA~1/MIE74D~1/RSERVE~1/R_SERVER\library\RevoScaleR\rxLibs\x64\hdfs.dll)
#Error in try(rxCall("RxHdfsConnect", params)) : 
  #System error: Attempting to load HDFS functions in unsupported environment. (Failed to load C:/PROGRA~1/MIE74D~1/RSERVE~1/R_SERVER\library\RevoScaleR\rxLibs\x64\hdfs.dll)
#In addition: 
#Warning message:
#The input is R dataframe. Run the job using local compute context instead. In order to parallelize execution in Spark, please convert it to composite XDF format first. 
```

**Comment by hongooi73** [comment]
TFS item 78051

**Comment by hongooi73** [comment]
Fixed in 9.2
